devp2p/tests/test_discovery.py

```diff
@@ -272,7 +277,7 @@ def test_bootstrap_udp():
     # now all nodes should know each other
     for i, app in enumerate(apps):
         num = len(kproto(app).routing)
-        print num
+        print(num)
         if i < len(apps) / 2:  # only the first half has enough time to get all updates
```

Comment:
Dunno if this should be replaced with `//` for floored division.  May not matter but this will cause the tests run differently in py2/py3 if `num` is odd.